### PR TITLE
Architecture doc

### PR DESCRIPTION
**Pull request by meganokeefe**
New doc: `_docs/architecture.md`. 

Still pretty minimal.

### CODE REVIEW

File: _docs/architecture.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+---
+
+**Last updated:** April 27, 2017
+
+The Sigma architecture was designed with simplicity and flexibility in mind, leveraging interfaces to swap in specific components
+as desired.
+
+At a high level, Sigma consists of a central manager, an API, a database, and a task runner.
+
+![sigma architecture diagram]({{ site.url }}/assets/arch_diagram.png)
```
Comment (mcaulfield) [inline _docs/architecture.md]: Diagram might need updating - API writes directly to DB and does not talk to the manager.

**Comment by adamschaub** [comment]
Did you mean to commit the `.sass-cache/<hash>/_layout.scssc` files? If not, let's remove them.

**Comment by stephenrlouie** [comment]
you should be able to remove those files under `.sass` jekyll generates some files when you run it locally

File: _docs/architecture.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+### Manager
+
+The Sigma Manager handles the internal logic of updating the services and operations. Its responsibilities include:
+- starting up new  or "inactive" services and service operations
+- checking existing "active" services and service operations for their progress, and updating accordingly
+- stopping service operations on timeout
+
+### Database
+
+Sigma has a generic database interface that specifies CRUDL operations on Plugins, Services, and ServiceOperations. Currently Sigma is using a [Couchbase](https://www.couchbase.com/) implementation of `database.go`. The database interface structure also allows for creating mocks and nullDBs for testing purposes.
```
Comment (stephenrlouie) [inline _docs/architecture.md]: Currently the team as implemented a couchbase implementation of the database interface..

idt the mocks and nullDB comment is necessary. By being an interface we get `mock` and we can definite any number of implementations just by being an interface.

**Pull request merged**